src/Driver/Connection.php

```diff
@@ -88,4 +88,7 @@ public function rollBack(): void;
      * @return resource|object
      */
     public function getNativeConnection();
+
+    /** @throws Exception */
+    public function getServerVersion(): string;
```

Comment:
This is needed for the following `catch` block to make sense: https://github.com/doctrine/dbal/blob/346ecec5dfcd99e1f5862668f853e4ee477ab761/src/Connection.php#L262-L266